### ISSUE TITLE
Can't configure source .. says GTK+ does not meet version requirements but it does

### ISSUE THREAD

**Issue by arcooke** [AmatCoder/mednaffe#30]
Hey.  I am trying to compile mednaffe from source on linux (Mint 17.3) .. but it is complaining that I need GTK >= 2.24.  I have 2.24.23, so it should work.. but it is still yelling at me.

Any ideas?  Thanks.

**dpkg -l libgtk\* | grep -e '^i' | grep -e 'libgtk-*[0-9]'** 

```
ii  libgtk-3-0:amd64                                            3.10.8~8+qiana                                    amd64        GTK+ graphical user interface library
ii  libgtk-3-bin                                                3.10.8~8+qiana                                    amd64        programs for the GTK+ graphical user interface library
ii  libgtk-3-common                                             3.10.8~8+qiana                                    all          common files for the GTK+ graphical user interface library
ii  libgtk2-imageview-perl                                      0.05-1build3                                      amd64        Perl bindings for the GtkImageView image viewer widget
ii  libgtk2-perl                                                2:1.249-2                                         amd64        Perl interface to the 2.x series of the Gimp Toolkit library
ii  libgtk2-unique-perl                                         0.05-1build3                                      amd64        module for single instance applications
ii  libgtk2.0-0:amd64                                           2.24.23-0ubuntu1.4                                amd64        GTK+ graphical user interface library
ii  libgtk2.0-0:i386                                            2.24.23-0ubuntu1.4                                i386         GTK+ graphical user interface library
ii  libgtk2.0-bin                                               2.24.23-0ubuntu1.4                                amd64        programs for the GTK+ graphical user interface library
ii  libgtk2.0-cil                                               2.12.10-5                                         amd64        CLI binding for the GTK+ toolkit 2.12
ii  libgtk2.0-common                                            2.24.23-0ubuntu1.4                                all          common files for the GTK+ graphical user interface library

```

**./configure** 

```
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking whether gcc understands -c and -o together... yes
checking for style of include used by make... GNU
checking dependency style of gcc... gcc3
checking for pkg-config... /usr/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for GTK... no
configure: error: Mednaffe needs GTK+ >= 2.24
```

**Comment by arcooke** [comment]
Nevermind you can close.. It was asking for dev libs.  

Installed libgtk2.0-dev and it worked

**Comment by AmatCoder** [comment]
Yes, error messages are not clear.
Fixed on 7167ed3 commit.
